packages/block-library/src/legacy-widget/edit/preview.js

```diff
@@ -0,0 +1,49 @@
+/**
+ * WordPress dependencies
+ */
+import { addQueryArgs } from '@wordpress/url';
+import { useRef, useState, useCallback, useEffect } from '@wordpress/element';
+import { Disabled } from '@wordpress/components';
+import { __ } from '@wordpress/i18n';
+
+const DEFAULT_HEIGHT = 300;
+
+export default function PreviewIframe( { idBase, instance, isVisible } ) {
+	const ref = useRef();
+
+	const [ height, setHeight ] = useState( DEFAULT_HEIGHT );
+
+	const adjustHeight = useCallback( () => {
+		setHeight( ref.current.contentDocument.body.scrollHeight );
+	} );
```

Comment:
Forgot the second argument?
```suggestion
	}, [] );
```